Common subdirectories: slstatus/components and suckless/slstatus/components
diff -u slstatus/config.def.h suckless/slstatus/config.def.h
--- slstatus/config.def.h	2024-11-22 14:17:01.381459476 +0100
+++ suckless/slstatus/config.def.h	2024-11-22 14:16:22.871243081 +0100
@@ -65,5 +65,10 @@
  */
 static const struct arg args[] = {
 	/* function format          argument */
-	{ datetime, "%s",           "%F %T" },
+	{ datetime,    "%s;",         "%d-%m-%Y" },
+	{ cpu_perc,    "%s% | ",      NULL },
+	{ ram_perc,    "%s% | ",      NULL },
+	{ vol_perc,    "%s",          "Master" },
+	{ keymap,      "% | %s | ",   NULL },
+	{ datetime,    "%s   ",       "%I:%M %p" },
 };
diff -u slstatus/config.mk suckless/slstatus/config.mk
--- slstatus/config.mk	2024-11-22 14:17:01.381459476 +0100
+++ suckless/slstatus/config.mk	2024-11-16 19:02:38.277431519 +0100
@@ -11,9 +11,9 @@
 X11LIB = /usr/X11R6/lib
 
 # flags
-CPPFLAGS = -I$(X11INC) -D_DEFAULT_SOURCE -DVERSION=\"${VERSION}\"
+CPPFLAGS = -I$(X11INC) -D_DEFAULT_SOURCE -DVERSION=\"${VERSION}\" -DALSA
 CFLAGS   = -std=c99 -pedantic -Wall -Wextra -Wno-unused-parameter -Os
-LDFLAGS  = -L$(X11LIB) -s
+LDFLAGS  = -L$(X11LIB) -s -lasound
 # OpenBSD: add -lsndio
 # FreeBSD: add -lkvm -lsndio
 LDLIBS   = -lX11
Only in slstatus/: .git
Only in slstatus/: LICENSE
Only in suckless/slstatus/: patches
Only in slstatus/: README
